mysql-test/suite/rocksdb/t/rocksdb_bug7883_2.test

```diff
@@ -0,0 +1,39 @@
+--source include/have_rocksdb.inc
+
+--connect(con1, localhost, root,,)
+--connect(con2, localhost, root,,)
+
+--connection con1
+SET @@global.rocksdb_flush_log_at_trx_commit = 2;
+SET @@session.rocksdb_write_disable_wal=ON;
+
+--connection con2
+SET @@session.rocksdb_write_disable_wal=OFF;
+SET @@global.rocksdb_flush_log_at_trx_commit = 1;
+
+--connection con1
+# Now rocksdb_flush_log_at_trx_commit = 1 and rocksdb_write_disable_wal=ON
+# The following statements have caused crash as
+# rocksdb_write_disable_wal=ON is incompatible with
+# synchronous writes (rocksdb_flush_log_at_trx_commit=1).
+# See https://jira.percona.com/browse/PS-7883 for details.
+
+CREATE TABLE tt_12 (ipkey INT AUTO_INCREMENT, i1 INT, d2 DOUBLE,
+                    PRIMARY KEY(ipkey), INDEX tt_12i0(d2, i1 ASC, ipkey),
+                    INDEX tt_12i1(ipkey, i1, d2))
+                   ROW_FORMAT=DYNAMIC ENGINE=RocksDB;
+
+REPLACE INTO tt_12  (ipkey, i1, d2) VALUES(2097, 6145, 0.00000);
+
+call mtr.add_suppression("Sync writes has to enable WAL. Switching to rocksdb_write_disable_wal = false for the time when sync writes are enabled.");
+ALTER TABLE tt_12 MODIFY COLUMN  ipkey INT(14), LOCK=DEFAULT, ALGORITHM=DEFAULT;
+
+--let $grep_file=$MYSQLTEST_VARDIR/log/mysqld.1.err
```

Comment:
Please add the check if rocksdb_write_disable_wal reports the proper value (false)